src/cascadia/TerminalSettingsEditor/ColorSchemes.xaml

```diff
@@ -194,9 +193,8 @@ the MIT License. See LICENSE in the project root for license information. -->
                           Grid.Column="1"
                           Style="{StaticResource ItemsControlStyle}">
                 <StackPanel Style="{StaticResource ColorStackPanelStyle}">
-                    <TextBlock x:Uid="ColorScheme_Foreground"
-                               Style="{StaticResource ColorHeaderStyle}"/>
-                    <Button Background="{Binding Color, ElementName=ForegroundPicker, Converter={StaticResource ColorToBrushConverter}, Mode=OneWay}">
+                    <Button x:Uid="ColorScheme_Foreground"

```

Comment:
@cinnamon-msft Do we want to keep the labels for Foreground, Background, Selection Color and Cursor Color?